WcaOnRails/config/application.rb

```diff
@@ -73,5 +73,8 @@ class Application < Rails::Application
     config.middleware.use Middlewares::WardenUserLogger, logger: ->(s) { Rails.logger.info(s) }
 
     config.autoload_paths << Rails.root.join('lib')
+
+    config.action_controller.asset_host = ENVied.ROOT_URL
+    Rails.application.routes.default_url_options[:host] = ENVied.ROOT_URL
```

Comment:
I made user/competition/person `url` to use `_url` instead of `_path` as well (makes sense, right?). We could explicitly specify the host parameter in every url helper call, but that's cleaner way I think.